### DISCUSSION TITLE
EQ5 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#77] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/EQ5.html
Author: @tdegeorge 
Primary Reviewer: @kathypinzon

**Comment by kathypinzon** [comment]
Def 1.5.1 - A quadratic equation "has the form"
where a, b, and c are real numbers with a \neq 0.

**Comment by kathypinzon** [comment]
Activity 1.5.2 - The form ax^2+bx+c=0 is called standard form.  Quadratic equations can come in many forms.  
Which of the following are quadratic equations?
Use "are" instead of is so that it leads them to the fact there are multiple answers.

**Comment by kathypinzon** [comment]
Def 1.5.3 instead of * use \cdot for multiplication.
In other words, in the real-numbers if a product is 0, then at least one of the numbers must be zero.

**Comment by kathypinzon** [comment]
Remark 1.5.5  you may want to expand on what "nicely" means. also why are there * around before? maybe make the word bold instead.

**Comment by kathypinzon** [comment]
Activity 1.5.4 (a) Which of the following equations are in the correct form to apply Definition 1.5.3 without any further manipulation?

**Comment by kathypinzon** [comment]
Activity 1.5.6 I would say "which is in standard form and not factored" to help reinforce those definitions.
(a) Find the factored form of ...
(b) Apply def 1.5.3 to find the solution of ...

**Comment by kathypinzon** [comment]
Def 1.5.9 i like x^2=k and x=\sqrt{k} better than using k^2.  Otherwise I feel like they only try to apply it when there is a perfect square there.

**Comment by kathypinzon** [comment]
1.5.10 part b I would say - rewrite the equation so that def 1.5.9 can be applied. Then give some MC options.
3x^{2}-8=4
3x^{2}=12
x^{2}=4
x=2

**Comment by kathypinzon** [comment]
I suggest moving the quadratic formula above completing the square since not all folks cover that....

**Comment by kathypinzon** [comment]
Def 1.5.13 "equals sign"

**Comment by kathypinzon** [comment]
Activity 1.5.14  I don't think you need the questions for part (b) and (c).  The sentences before them are enough.

**Comment by kathypinzon** [comment]
Activity 1.5.16 (a) should say variable x "are" on one side.
(c) the wording seems strange...
(d) do you want the answer here to be 8 or 4? if you want 4, then say what can we add inside the parentheses.

**Comment by kathypinzon** [comment]
Activity 1.5.19 I would include MC options for part (a)
part (c) i think it's fine to just say solution - since it could be consider the solution set so is a singular term even if there are 2 values.

**Comment by kathypinzon** [comment]
Activity 1.5.20 same comments as above for 1.5.19

**Comment by kathypinzon** [comment]
Activity 1.5.23 
For the introduction statement - I would phrase it like
"As you have seen there are any different ways to solve a quadratic equation, but not every method works or is the best approach to every problem.  It is useful to identify patterns for when each method works best."
